flutter_secure_storage/ios/Classes/SwiftFlutterSecureStoragePlugin.swift

```diff
@@ -6,15 +6,20 @@
 //
 
 import Flutter
+import UIKit
 
-public class SwiftFlutterSecureStoragePlugin: NSObject, FlutterPlugin {
+public class SwiftFlutterSecureStoragePlugin: NSObject, FlutterPlugin, FlutterStreamHandler {
     
     private let flutterSecureStorageManager: FlutterSecureStorage = FlutterSecureStorage()
-    
+    private var secStoreAvailabilitySink: FlutterEventSink?
+
     public static func register(with registrar: FlutterPluginRegistrar) {
         let channel = FlutterMethodChannel(name: "plugins.it_nomads.com/flutter_secure_storage", binaryMessenger: registrar.messenger())
+        let eventChannel = FlutterEventChannel("plugins.it_nomads.com/flutter_secure_storage/events", binaryMessenger: registrar.messenger())
```

Comment:
This fails for me on the swift compiler, because it is missing the `name:` prefix. This fixes it:

```suggestion
        let eventChannel = FlutterEventChannel(name: "plugins.it_nomads.com/flutter_secure_storage/events", binaryMessenger: registrar.messenger())
```

Reply:
Oh yeah right. Thanks for the fix.